hack/fetch-all-latest-and-push.sh

```diff
@@ -43,6 +43,7 @@ repos=(
     cluster-bootstrap
     code-generator
     component-base
+    cri-api
```

Comment:
we should really make this consume the rules file. Or just move it over completely into the main publisher process. We maybe just lack a clone command. We fetches changes already.

Reply:
Created https://github.com/kubernetes/publishing-bot/issues/171